wizard_builder/managers.py

```diff
@@ -12,22 +15,29 @@ def on_site(self, site_id=None):
         )
 
 
-class PageBaseManager(InheritanceManager):
+class FormQuestionQuerySet(InheritanceQuerySet):
+    pass
 
-    def get_queryset(self):
-        base_queryset = PageBaseQuerySet(self.model, using=self._db)
-        return base_queryset.select_subclasses()
 
-    def on_site(self, site_id=None):
-        return self.get_queryset().on_site(site_id)
+class AutoDowncastingManager(InheritanceManager):
 
+    def get_queryset(self):
```

Comment:
ok so like in a sentence...this dumps the parent class if it's run from `dumpdata`, and the subclasses if not?